pset-iterators.js

```diff
@@ -43,8 +43,16 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+const firstWhigPresident = presidents.find(function (president) {
+  return president.party === 'Whig'
+})
+
+console.log("First president from the Whig party:",firstWhigPresident.president)
```

Comment:
@AtleLohrmann Good Job here using the find method to return the first president from the Whig party.

Take a look at this solution [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L35)